test/bookmarks.js

```diff
@@ -29,6 +29,15 @@ describe('/bookmarks', () => {
     res.body.should.be.a('object');
   });
 
+  it('should return status code 40 when user is not logged in', async () => {
+    const res = await chai
+      .request(app)
+      .get('/api/v1/articles/bookmarks/search?q=this-is-trd-post-title-u87ddsa')
+      .set('access-token', userToken);
+    res.status.should.equal(200);
```

Comment:
When a user is not signed in, status shouldn't be 200

Reply:
this is not from me

Reply:
@el-Joft, I think he was trying to set access-token to fakeToken and status code to 401, see if you can fix this.

Reply:
I will when next I am pushing